streamer/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = '0.6.0'
+__version__ = '0.5.1'  # x-release-please-version
```

Comment:
why not start from a later version?

Reply:
0.5.1 was the latest we released.  The release tool (release-please) expects things to be synced (GitHub releases API, numbers in source, etc).  After I merge this, release-please will create a release PR that updates this to 1.0.0.